po/de.po

```diff
@@ -13746,12 +13744,12 @@ msgstr ""
 
 #: ../src/iop/retouch.c:2235
 msgid "type for the blur algorithm"
-msgstr "Typ des Weichzeichnungsalgorithmus"
+msgstr "Typ des Weichzeichner-Filters"
```

Comment:
```suggestion
msgstr "Typ des Weichzeichnungsfilters"
```
This seems to be a proper term, it is used in academic literature and also in photoshop :stuck_out_tongue: